front/style/mixins/_colors.scss

```diff
@@ -26,6 +26,7 @@ $yellow-hover: #c4d02a; //old
 $yellow--light: #FFEB00;
 $cta: #1254d8;
 $cta-hover: #0251C0;
+$white: #FFFFFF;
```

Comment:
duplicate entry

Reply:
For `$white`, I would just use `white` in the code. That's what we've been doing so far I think. That's just my opinion 😄

Reply:
@marineb Yes but if we decide to pass all the Website's white in #FEFEFE for example.
It will be more maintainable to change the $white variable than to change all class that have color: white, right?

Reply:
I don't know for a value of $white, but not need to add new `$white` variable if this already exists